cli/src/semgrep/env.py

```diff
@@ -50,6 +50,13 @@ class Env:
     between multiple invocations.
     """
 
+    fail_open_url: str = field(
+        default=EnvFactory(
+            ["FAIL_OPEN_URL", "SEMGREP_FAIL_OPEN_URL"],
+            "https://fail-open.semgrep.dev/failure",
```

Comment:
Either we need to update the TF module or this needs to be configured to `https://fail-open.prod.semgrep.dev` since the TF module always adds the env name.

Reply:
Updating to `https://fail-open.prod.semgrep.dev/failure`